Bottom: e89b0b24b5c92bd754bf00b5a9e35d5c6c9858e6
Top:    505e48e897dfa669e3516bd891cbea107d7238d9
Author: Christopher Voltz <cjunk@voltz.ws>
Date:   2017-05-01 23:01:04 -0500

Refresh of rakefile

---

diff --git a/Rakefile b/Rakefile
index 8b91519..fa29060 100644
--- a/Rakefile
+++ b/Rakefile
@@ -2,3 +2,10 @@ desc "Remove trailing whitespace"
 task :rm_whitespace do
   `sed -i 's/[[:space:]]*$//' script.js syntax.php`
 end
+
+desc "Reformat JS code"
+task :reformat_js do
+  mv "script.js", "script.js.input"
+  `uglifyjs --beautify --output script.js -- script.js.input`
+  rm "script.js.input"
+end
